src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import React from 'react';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={user.email}>
```

Comment:
that's what I see when I click on this link, use `mailto:` link
![image](https://user-images.githubusercontent.com/37161315/180390305-14fc17a3-d1d5-4550-bf14-df7637f2ec50.png)